README.md

```diff
@@ -91,7 +91,13 @@ Note that `base_path` just modifies the links.
 Get the latest version of the given human or canonical page name:
 
 ```ruby
-page = wiki.page('page-name')
+page = wiki.page('/page name') # Finds pages in the root directory of the wiki that are named 'page name' with a valid extension. 
+# => <Gollum::Page>
+
+page = wiki.page('page name') # For convenience, you can leave out the '/' in front. Paths are assumed to be relative to '/'.
```

Comment:
> Paths are assumed to be relative to '/'.

Is this still correct?

Reply:
Yes -- `wiki.page('Bilbo')` looks up `/Bilbo`